Makefile

```diff
@@ -27,6 +27,9 @@ flow:
 fix:
 	./node_modules/.bin/eslint packages/ --format=codeframe --fix
 
+prettify:
+	./node_modules/.bin/prettier --trailing-comma all --write "packages/**/src/*.js"
```

Comment:
Got lost when commas were changed to `all`. Feeling like we should include `--print-width 110`?